tests/ui/specs/TooltipDecorator/TooltipDecorator-specs.js

```diff
@@ -0,0 +1,50 @@
+const Page = require('./TooltipDecoratorPage');
+
+describe('TooltipDecorator', function () {
+
+	beforeEach(function () {
+		Page.open();
+	});
+
+	const {
+		tooltipDefault,
```

Comment:
tooltipDefaultButton or tooltipButtonDefault is better.
It is confused with the tooltip.